src/compiler/scala/tools/nsc/typechecker/NamesDefaults.scala

```diff
@@ -118,7 +118,7 @@ trait NamesDefaults { self: Analyzer =>
    * @param pt the expected type for calling typer.doTypedApply
    *
    * @param tree: the function application tree
-   * @argPos: a function mapping arguments from their current position to the
+   * @param argPos: a function mapping arguments from their current position to the
```

Comment:
Is it possible I'm looking at the before and not the after? https://ashawley.github.io/scala/2.13.0/compiler/scala/tools/nsc/typechecker/NamesDefaults.html#transformNamedApplication(typer:NamesDefaults.this.Typer,mode:scala.tools.nsc.Mode,pt:NamesDefaults.this.global.Type)(tree:NamesDefaults.this.global.Tree,argPos:Int=%3EInt):NamesDefaults.this.global.Tree

Reply:
I'm seeing the same issue, locally.  I wonder if it's the blank line on line 119?

Reply:
Actually, it's the colons...

Reply:
People should go in to have their colons checked per their doctor's recommendation.

Reply:
I've fixed the colons.